python/cudf/cudf/_libxx/io/io_types.pxd

```diff
@@ -22,14 +22,14 @@ cdef extern from "cudf/io/types.h" \
         QUOTE_NONE,
 
     ctypedef enum compression_type:
-        NONE,
-        AUTO,
-        SNAPPY,
-        GZIP,
-        BZIP2,
-        BROTLI,
-        ZIP,
-        XZ,
+        none "cudf::experimental::io::compression_type::NONE"
```

Comment:
Any reason for changing from caps to lowercase?

Reply:
no logical reason, I will change them back